server/prisma/schema.prisma

```diff
@@ -12,3 +12,77 @@ datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
+
+model User {
+  zid         String   @id
+  description String?
+  degree      String?
+  year        Int?
+  verified    Boolean
+  admin       Boolean  @default(false)
+  createdAt   DateTime @default(now())
+  updatedAt   DateTime @updatedAt
+
+  reviews Review[]
+}
+
+model Society {
+  id            String   @id @default(uuid())
+  name          String   @unique
+  description   String
+  icon          String   @default("https://kansai-resilience-forum.jp/wp-content/uploads/2019/02/IAFOR-Blank-Avatar-Image-1.jpg")
+  facebookUrl   String?
+  discordUrl    String?
+  emailUrl      String?
+  averageRating Float    @default(0)
+  totalReviews  Int      @default(0)
+  createdAt     DateTime @default(now())
+  updatedAt     DateTime @updatedAt
+
+  reviews Review[]
+  tags    TagsOnSocieties[]
+}
+
+model Review {
+  id        String   @id @default(uuid())
+  title     String?
+  content   String?
+  rating    Int
+  anonymous Boolean
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  userId    String
+  user      User            @relation(fields: [userId], references: [zid])
```

Comment:
what does this @relation fields and references mean again?

Reply:
A foreign key - saying that the Review has a relationship with a User (the User object), and the @relation just says they are "linked" on the Review's userId and the User's zid.